docs/commands/rhoas_service-registry_describe.md

```diff
@@ -28,7 +28,7 @@ rhoas service-registry describe --id 1iSY6RQ3JKI8Q0OTmjQFd3ocFRg
 ```
       --id string       Unique ID of the Service Registry instance (if not provided, the current Service Registry instance will be used)
       --name string     Name of the Service Registry instance to view
-  -o, --output string   Format in which to display the Service Registry instance (choose from: "json", "yml", "yaml" or "none") (default "json")
+  -o, --output string   Format in which to display the Service Registry instance (choose from: "json", "yml", "yaml") (default "json")
```

Comment:
Missing none

Reply:
Hi, I actually manually edited all to reflect none in my previous commit, but in this one, I used the generate doc option and it skipped none for a couple of these files, and when i checked each manually, from a logical standpoint it made sense to omit the none option, for commands like "describe"
I wanted to get your input on this before reverting the auto generated docs to reflect my own change.